File: sdks/php/src/Types/BillableMetricResponseAggregationType.php
<?php

namespace NovaBilling\Types;

enum BillableMetricResponseAggregationType: string
{
    case Count = "COUNT";
    case Sum = "SUM";
    case Max = "MAX";
    case UniqueCount = "UNIQUE_COUNT";
    case Latest = "LATEST";
    case WeightedSum = "WEIGHTED_SUM";
}

File: sdks/ruby/lib/novabilling/types/billable_metric_response_aggregation_type.rb
# frozen_string_literal: true

module Novabilling
  module Types
    module BillableMetricResponseAggregationType
      extend Novabilling::Internal::Types::Enum

      COUNT = "COUNT"
      SUM = "SUM"
      MAX = "MAX"
      UNIQUE_COUNT = "UNIQUE_COUNT"
      LATEST = "LATEST"
      WEIGHTED_SUM = "WEIGHTED_SUM"
    end
  end
end
